examples/eth1_add_route_rule.yml

```diff
@@ -0,0 +1,46 @@
+---
+interfaces:
+  - name: eth1
```

Comment:
The file is named based on this iface, but now two are used.
I would just name the file `add_route_rules.yml`.